Code/Tools/SceneAPI/SceneCore/Import/ManifestImportRequestHandler.h

```diff
@@ -17,22 +17,24 @@ namespace AZ
     {
         namespace Import
         {
-            class ManifestImportRequestHandler
+            class SCENE_CORE_CLASS ManifestImportRequestHandler
```

Comment:
Exporting the class should export all of it's member functions.
The [__attribute__ ((visibility ("default")))](https://gcc.gnu.org/wiki/Visibility) option of GCC/Clang does that which is what the SCENE_CORE_CLASS macro is defined to.

Therefore the individual tagging the individual member functions with `SCENE_CORE_API` isn't needed.

Reply:
I get compile errors without SCENE_CORE_API, and no compile errors with it, so it sure _seems_ like it's needed...